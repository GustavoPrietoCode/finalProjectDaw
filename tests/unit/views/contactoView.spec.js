import { shallowMount } from '@vue/test-utils'
import Contacto from '@/views/ContactoView'

describe('Pruebas About View', () => {

    let wrapper

    beforeEach(() => {
        wrapper = shallowMount ( Contacto )
    })

    test('Debe de hacer match con el snapshot', () =>{

        expect( wrapper.html() ).toMatchSnapshot()

    })


})