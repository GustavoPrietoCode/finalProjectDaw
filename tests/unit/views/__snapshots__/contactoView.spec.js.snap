// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas About View Debe de hacer match con el snapshot 1`] = `
<div class="about">
  <h1>Esta es la p√°gina de contacto</h1>
</div>
`;
